README.md

```diff
@@ -1,25 +1,34 @@
-# scorper  ![Build Status](https://github.com/bung87/scorper/workflows/Test/badge.svg)  
+# SCORPER  ![Build Status](https://github.com/bung87/scorper/workflows/Test/badge.svg)  
 
 ![scorper](artwork/scorper.png)  
 
 
 [travis]: https://travis-ci.org/bung87/scorper.svg?branch=master
 
-scorper is a micro and elegant web framework written in Nim  
+**Elegant, performant, asynchronous micro web framework written in Nim**  
 
-Build upon [chronos](https://github.com/status-im/nim-chronos.git) and serveral excellent projects.
+Built using [chronos](https://github.com/status-im/nim-chronos.git) and other powerful libraries.
 
-`scorper` will self contain manageable dependencies source code as much as posibble for accelerating development.  
 
-**Notice**: `scorper` heavily relys on `chronos` which use its own `async` macro it will conflicts with std `asyncdispatch`'s `async` macro, check your project dependencies chronos support first.    
+**Note**: `scorper` is built using `chronos` which implements an `async`/`await` paradigm that is, at its core, different and therefore incompatible with the nim std library `asyncdispatch`'s `async`; this conflict is irresolvable and you will have to ensure your project dependencies support chronos async/await (note: many libraries do offer this, often using a compiler switch).     
 
-## Production 
+> `scorper` will self contain manageable dependency source code for accelerated development.  
 
-compile your program with `-d:chronosStrictException` if you dont want any exception crash your program.  for details check chronos's readme [#Error handling](https://github.com/status-im/nim-chronos#error-handling)
+## Production
 
-## Compile flags  
-`-d:ssl`  
-then use `newScorper` proc and pass `isSecurity = true`,`privateKey`, `certificate`  
+By default, exceptions will be raised and exit the program.
+
+You can disable this with the compile switch `-d:chronosStrictException`. See chronos [#Error handling](https://github.com/status-im/nim-chronos#error-handling) for more details.
```

Comment:
this flag does not disable the behavior, it just do strict check and log out where will cause exception trigger the behavior.

Reply:
Ah I think I understand now; I hadn't actually looked at the chronos documentation for the exception handling before. Good to know now XD; I've updated the readme to just include a few snippets from their document and then moved it down the readme.